test/k_module_registry.js

```diff
@@ -133,6 +138,57 @@ contract("ModuleRegistry", accounts => {
         `);
     });
 
+    describe("Test the initialize the function", async () => {
+        it("Should successfully update the implementation address -- fail because polymathRegistry address is 0x", async () => {
+            let bytesProxy = encodeProxyCall(MRProxyParameters, [
+                "0x0000000000000000000000000000000000000000",
```

Comment:
For cleaner code, maybe let's define `const nullAddress = "0x0000000000000000000000000000000000000000"` and use it everywhere. It is used a lot in all the test cases.